omero/conf_autogen.py

```diff
@@ -1,4 +1,5 @@
 omero_db_version = "OMERO5.4DEV"
 omero_db_patch = "3"
+versions_bioformats = "5.6.0"
```

Comment:
Could that value be extracted from ``omero.properties``?

Reply:
I thought that was what Seb had done

Reply:
done in ``autogen_db_version.py``

Reply:
@sbesson I'll leave this one to you

Reply:
Isn't that done in c132c4f0cc2244df1019a7b87f104e45a0d447f6? See  the commit message of 741450ce83dd4bef349ecb5fc7981529a948ef5d

Reply:
@hflynn sorry that's what I meant by done in ``autogen_db_version.py``
I first missed it